src/main/java/com/shaft/gui/element/ElementActions.java

```diff
@@ -330,43 +329,43 @@ static By updateLocatorWithAIGenratedOne(By elementLocator) {
         return elementLocator;
     }
 
-    private static boolean identifyUniqueElement(WebDriver driver, By elementLocator, int numberOfAttempts,
+    private static boolean identifyUniqueElement(WebDriver driver, By elementLocator,
```

Comment:
Codacy found an issue: [The method 'identifyUniqueElement(WebDriver, By, boolean)' has an NPath complexity of 2147483647](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5734119)